src/src/_lib/index.ts

```diff
@@ -0,0 +1,69 @@
+import {DateTime, Interval} from 'luxon'
+
+import {CalendarSelectionRangeI} from '../types'
```

Comment:
did you rename the file or create a new one with a `ts` extension?

Reply:
Before we did not have any `types`, its a new file.